src/PKSim.Core/Mappers/ProcessToProcessBuilderMapper.cs

```diff
@@ -286,26 +316,60 @@ public TransporterMoleculeContainer ActiveTransportFrom(CompoundProcess compound
          return transporterMoleculeContainer;
       }
 
-      // private void updateTransporterTagsFor(IndividualTransporter transporter, ITransportBuilder transporterBuilder, string simulationProcessName)
-      // {
-      //    //if one organ was specified already, no need to create the list of not tags!
-      //    var allOrgans = transporter.AllOrgansWhereProcessTakesPlace(simulationProcessName).ToList();
-      //    var allMatchTags = transporterBuilder.SourceCriteria.OfType<MatchTagCondition>()
-      //       .Select(x => x.Tag);
-      //
-      //    if (allMatchTags.Intersect(allOrgans).Any())
-      //       return;
-      //
-      //    foreach (var organName in transporter.AllOrgansWhereProcessDoesNotTakePlace(simulationProcessName))
-      //    {
-      //       transporterBuilder.SourceCriteria.Add(new NotMatchTagCondition(organName));
-      //    }
-      // }
-
-      private ITransportBuilder activeTransportFrom(CompoundProcess process, string individualProcessName, IFormulaCache formulaCache)
+      private IReadOnlyCollection<InducedProcess> allInducedProcessesFor(IndividualTransporter transporter, Individual individual)
+      {
+         var inducedProcessCache = new Cache<string, InducedProcess>(x => x.Name, x => null);
+         var allTransporterContainers = individual.AllMoleculeContainersFor<TransporterExpressionContainer>(transporter)
+            .Where(x => x.TransportDirection != TransportDirectionId.None).ToList();
+
+         //Global containers surrogate do not have a logical container associated
+         var allOrganNames = new HashSet<string>(allTransporterContainers.Select(x => x.LogicalContainerName).Where(x => !string.IsNullOrEmpty(x)));
+
+         foreach (var transporterContainer in allTransporterContainers)
+         {
+            var allTransportTemplates = _transportTemplateRepository.All()
+               .Where(x => x.TransportDirection == transporterContainer.TransportDirection)
+               //For apical transport for Lumen => Mucosa, the compartment is the name of our mapping
+               .Where(x => (x.SourceOrgan == transporterContainer.LogicalContainerName ||
+                            x.SourceCompartment == transporterContainer.LogicalContainerName));
+
+            foreach (var transportTemplate in allTransportTemplates)
+            {
+               var inducedProcess = inducedProcessCache[transportTemplate.ProcessName];
+               if (inducedProcess == null)
+               {
+                  inducedProcess = new InducedProcess {Name = transportTemplate.ProcessName};
+                  inducedProcess.AddOrgansToExclude(allOrganNames);
+                  inducedProcessCache.Add(inducedProcess);
+               }
+
+               inducedProcess.AddSourceOrgan(transportTemplate.SourceOrgan);
```

Comment:
I am not even sure if we need this. Keeping for now but the transports are well defined already. Either everywhere or specific and in that case, source etc.. comes from DB